test/run-tests

```diff
@@ -85,13 +85,19 @@ def run(*argv, env=None, check=True):
     Small wrapper around subprocess.run(), which prints the command to be
     executed and raises an exception by default.
     """
+    runenv = dict(os.environ)
     if env:
         envrepr = ' '.join(["=".join(e) for e in env.items()]) + ' '
     else:
         envrepr = ''
 
+    if env:
+        runenv.update(env)
+
     print('+ ' + envrepr + ' '.join(shlex.quote(a) for a in argv))
```

Comment:
This is intentional, it prints the extra env that the test runner adds to the default env. Printing the whole env would be too much and might be even insecure since it might also contain secrets.